src/assets/locales/fr.json

```diff
@@ -217,74 +244,80 @@
   "media-browser.report-issue": "Signaler une erreur",
-  "media-browser.search-placeholder.avatars": "Chercher des Avatars...",
+  "media-browser.search-placeholder.avatars": "Chercher des avatars...",
   "media-browser.search-placeholder.default": "Chercher...",
   "media-browser.search-placeholder.gifs": "Chercher des GIFs...",
-  "media-browser.search-placeholder.images": "Chercher des Images...",
-  "media-browser.search-placeholder.poly": "Chercher des modèles Google Poly...",
-  "media-browser.search-placeholder.scenes": "Chercher des Scènes...",
+  "media-browser.search-placeholder.images": "Chercher des images...",
+  "media-browser.search-placeholder.scenes": "Chercher des scènes...",
   "media-browser.search-placeholder.sketchfab": "Chercher des modèles Sketchfab...",
   "media-browser.search-placeholder.twitch": "Chercher des streams Twitch...",
-  "media-browser.search-placeholder.videos": "Chercher des Videos...",
-  "media-browser.search-placeholder.youtube": "Chercher des videos Youtube...",
+  "media-browser.search-placeholder.videos": "Chercher des vidéos...",
+  "media-browser.search-placeholder.youtube": "Chercher des vidéos YouTube...",
   "media-tile.action.copy-avatar": "Copier vers mes avatars",
-  "media-tile.action.copy-scene": "Copier vers mes scenes",
+  "media-tile.action.copy-scene": "Copier vers mes scènes",
   "media-tile.action.edit-avatar": "Éditer l'avatar",
   "media-tile.action.edit-scene": "Éditer la scène",
-  "media-tile.action.room-info": "Info de la Salle",
+  "media-tile.action.room-info": "Info de la salle",
   "media-tile.action.show-similar-avatars": "Montrer les avatars similaires",
-  "media-tile.joined-room": "Joint {relativeTime}",
+  "media-tile.joined-room": "A rejoint {relativeTime}",
   "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au micro",
   "mic-permissions-modal.mic-access-needed": "Vous avez besoin d'un micro pour être entendu par les autres participants.{linebreak}Vous pourrez couper votre micro dans la salle.",
   "mic-permissions-modal.title": "Entrer par un appareil",
   "mic-setup-modal.check-mic": "Vérifiez votre micro et l'audio avant d'entrer.",
-  "mic-setup-modal.enter-room-button": "Entrer dans la Salle",
+  "mic-setup-modal.enter-room-button": "Entrer dans la salle",
   "mic-setup-modal.mic-disabled": "Micro Désactivé",
   "mic-setup-modal.mute-mic-toggle": "Couper mon micro",
   "mic-setup-modal.test-audio": "Cliquer pour Tester l'Audio",
   "mic-setup-modal.test-mic": "Parler pour Tester le micro",
   "mic-setup-modal.title": "Réglages du micro",
   "more-menu-popover.title": "Plus",
-  "more-menu.close-room": "Fermer la Salle",
+  "more-menu.close-room": "Fermer la salle",
   "more-menu.community": "Communauté",
-  "more-menu.controls": "Controles",
-  "more-menu.create-room": "Créer une Salle",
-  "more-menu.enter-leave-room": "Quitter la Salle",
+  "more-menu.controls": "Contrôles",
```

Comment:
"Commandes" ("Controls" in english != "Contrôles" in french)

Reply:
ah ah I think you're right on this one. Ok I'll change to "Commandes".